business/workloads.go

```diff
@@ -28,6 +29,30 @@ type WorkloadService struct {
 	businessLayer *Layer
 }
 
+var (
+	ew                sync.RWMutex
+	excludedWorkloads map[string]bool
+)
+
+func isExcludedWorkloadsEmpty() bool {
+	defer ew.RUnlock()
+	ew.RLock()
+	return excludedWorkloads == nil
+}
+
+func setExcludedWorkloads(m map[string]bool) {
+	defer ew.Unlock()
+	ew.Lock()
+	excludedWorkloads = m
+}
+
+func isWorkloadIncluded(workload string) bool {
+	if excludedWorkloads == nil {
```

Comment:
Why is ``isExcludedWorkloadsEmpty`` using mutex to protect ``excludedWorkloads == nil``, but this one isn't? And why isn't it just calling that method first? Actually, the ``!excludedWorkloads[workload]`` is not protected by the mutex also, it could have issues with ``setExcludedWorkloads``

Reply:
Indeed, this mutex probably it's not necessary at all, as excludedWorkloads is a one time initialization, similar as kialiCache.
I guess originally it was placed in a different code and that case introduced the mutex.
I'll review & check.